app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CalendarDao.java

```diff
@@ -14,8 +14,8 @@
     @Query("SELECT c.* FROM calendar c WHERE status != 'CANCEL'")
     List<CalendarItem> getAllNotCancelled();
 
-    @Query("SELECT c.* FROM calendar c WHERE dtstart LIKE '%' || :date || '%' AND status != 'CANCEL' ORDER BY dtstart ASC")
-    List<CalendarItem> getAllByDateNotCancelled(String date);
+    @Query("SELECT c.* FROM calendar c WHERE dtstart LIKE '%' || :date || '%' ORDER BY dtstart ASC")
+    List<CalendarItem> getAllByDate(String date);
```

Comment:
Silently swallowing events wasn't a great idea in the first place :see_no_evil:. Thanks for fixing #635